src/utils/persistent.ts

```diff
@@ -0,0 +1,23 @@
+class Persistent {
+
+    addresses = {
+        user: "",
+        multicall: "",
+        dao: ""
+    }
+
+    setAddresses(newAddresses) {
+
+        this.addresses = {
+            ...this.addresses,
+            ...newAddresses
+        }
+        window["TASKS"]?.map(t => t.instance.current.onAddressesUpdated());
+        window["MENU"]?.forceUpdate();
+        window["DAO"]?.onAddressesUpdated();
+
+    }
+
+}
+
+window["PERSISTENT"] = new Persistent();
```

Comment:
Firefox worked fine, but I got following error using Brave:
`Uncaught TypeError: Cannot assign to read only property 'PERSISTENT' of object '#<Window>'`
and `window.PERSISTENT` evaluates to `1` in the console. That attribute name might reserved by the browser for other things?

Reply:
You would be correct!

Found this: https://www.tabnine.com/code/javascript/functions/builtins/Window/PERSISTENT
In all caps too, how weird.

Guess i will change the naming